### PR TITLE
add Flask-GraphQL to conda-forge

### PR DESCRIPTION
**Pull request by rluria14**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@adbeda @agcopenhaver do you agree to be maintainers?

**Comment by adbeda** [comment]
Yes. happy to be a co-maintainer.

File: recipes/dash-cytoscape/meta.yaml [hunk 1]
```diff
@@ -1,40 +0,0 @@
-{% set name = "dash_cytoscape" %}
-{% set version = "0.1.1" %}
-
```
Comment (bgruening) [inline recipes/dash-cytoscape/meta.yaml]: Please do not remove this package ...

Comment (rluria14) [reply recipes/dash-cytoscape/meta.yaml]: Thanks for catching this. I had an error when pushing to my staged-recipes. Fixed.

**Comment by agcopenhaver** [comment]
I agree to be a maintainer

**Pull request closed without merging**